src/EnergyPlus/HVACManager.cc

```diff
@@ -2894,16 +2902,16 @@ void ReportAirHeatBalance(EnergyPlusData &state)
         ZnAirRpt(ZoneLoop).SysInletMass = 0;
         ZnAirRpt(ZoneLoop).SysOutletMass = 0;
         if (!ZoneEquipConfig(ZoneLoop).IsControlled) {
-            for (int j = 1; j <= ZoneEquipConfig(ZoneLoop).NumInletNodes; ++j) {
-                ZnAirRpt(ZoneLoop).SysInletMass += state.dataLoopNodes->Node(ZoneEquipConfig(ZoneLoop).InletNode(j)).MassFlowRate * TimeStepSys *
+            for (int k = 1; k <= ZoneEquipConfig(ZoneLoop).NumInletNodes; ++k) {
```

Comment:
These j index variables were shadowing a different j variable, so just moved them to k (almost moved them to range-based-loops but I know those are always appreciated)